src/service/mpris.ts

```diff
@@ -137,18 +144,50 @@ class MprisPlayer extends GObject.Object {
     }
 
     _cacheCoverArt() {
-        const { trackCoverUrl, coverPath } = this;
-        if (trackCoverUrl === '') {
+        if (this.trackCoverUrl === '') {
```

Comment:
it should be
```js
if (!this.trackCoverUrl)
  return
```

and mark coverPath as optional

Reply:
If we don't set this.coverPath here if we go from song with cover to song without cover it will continue to have the old cover showing instead of a blank/no cover.